configure_easybuild

```diff
@@ -26,7 +26,7 @@ fi
 # note: filtering Bison may break some installations, like Qt5 (see https://github.com/EESSI/software-layer/issues/49)
 # filtering pkg-config breaks R-bundle-Bioconductor installation (see also https://github.com/easybuilders/easybuild-easyconfigs/pull/11104)
 # problems occur when filtering pkg-config with gnuplot too (picks up Lua 5.1 from $EPREFIX rather than from Lua 5.3 dependency)
-DEPS_TO_FILTER=Autoconf,Automake,Autotools,binutils,bzip2,cURL,DBus,flex,gettext,gperf,help2man,intltool,libreadline,libtool,Lua,M4,makeinfo,ncurses,util-linux,XZ,zlib
+DEPS_TO_FILTER=Autoconf,Automake,Autotools,binutils,bzip2,cURL,DBus,flex,gettext,gperf,help2man,intltool,libreadline,libtool,Lua,M4,makeinfo,ncurses,OpenSSL,util-linux,XZ,zlib
```

Comment:
We shouldn't filter `OpenSSL` anymore for 2023.06 (which has OpenSSL 1.1.1), we only did that in the `2023.04` EESSI pilot because there we had OpenSSL 3.x

Reply:
Done. (We haven't actually used it when building for NESSI/2023.06, so likely just a glitch when merging upstream changes from EESSI into the PR.)

Reply:
`OpenSSL` was only added to `DEPS_TO_FILTER` in the `2023.04` branch, there could have been some mixup there, indeed...